setup.py

```diff
@@ -1,103 +1,21 @@
-        build_ext.run(self)
-
-
-
-compile_mac_options =  ["-mmacosx-version-min=10.15"] if sys.platform == 'darwin' else []
-my_c_module = setuptools.Extension('graph2tac.loader.clib.loader',
-                                   sources=[
-                                       f'graph2tac/loader/clib/graph_api_v{CAPNP_V}.capnp.cpp',
-                                       'graph2tac/loader/clib/loader.cpp',
-                                   ],
-                                   include_dirs=[numpy.get_include()],
-                                   extra_compile_args=(
-                                       [
-                                       "-std=c++17",
-                                       "-O3",
-                                       "-I/opt/local/include",
-                                       f"-DCAPNP_V={CAPNP_V}"
-                                        ] +
-                                       compile_mac_options
-                                       ),
-                                   extra_link_args=[
-                                       "-lkj",
-                                       "-lcapnp",
-                                       "-lcapnp-rpc",
-#                                       "-lstdc++fs",
-#                                      "-ltbb",         # we may need threads library but ok without it in current code
-                                       "-L/opt/local/lib",
-                                       ]
-                                   )
-
-# the reason for the static linking to stdc++fs library is
-# a necessety to run this software on a legacy linux system that misses GLIBCXX_3.4.26
-# very helpful explanation was given at
-# https://stackoverflow.com/questions/63902528/program-crashes-when-filesystempath-is-destroyed
-
-
 setup(
     name='graph2tac',
     packages=find_packages(),  # find all packages in the project instead of listing them 1-by-1
-    cmdclass={'build_ext': BuildCapnpFiles},
-    ext_modules=[my_c_module],
     version='0.1.0',
     description='graph2tac converts graphs to actions',
     author=' Mirek Olsak, Vasily Pestun, Jason Rute, Fidel I. Schaposnik Massolo',
     python_requires='>=3.9',
     include_package_data=True,
-    package_data={'graph2tac.loader': [f'clib/graph_api_v{CAPNP_V}.capnp']},
     entry_points={'console_scripts':
                   [
-                      'g2t-preprocess=graph2tac.loader.preprocess:main',
                       'g2t-train=graph2tac.tf2.train:main',
                       'g2t-train-tfgnn=graph2tac.tfgnn.train:main',
-                      'g2t-stat=graph2tac.loader.graph_stat:main',
-                      'g2t-printlog=graph2tac.printlog:main',
-                      'g2t-unpack=graph2tac.loader.unpack:main',
-                      'g2t-server=graph2tac.loader.predict_server:main',
                       'g2t-py-server=graph2tac.loader.py_predict_server:main',
```

Comment:
Should we just go back to calling this `g2t-server`?

Reply:
fine, then we could also remove the prefix from py_..._server

Reply:
I'm not against this, but as a warning: Renaming the file might get you some nasty conflicts in other branches. Better to wait with renaming the file until everything is merged.

Reply:
I see because git will think it is the original file... I think my `split_options2` will be fine, they have already branched from `simplify`. So the question is about @jasonrute 's WIP (about definition recalculation) which we are not sure if / when we are actually going to merge. Perhaps, Jason, you could first merge `simplify` into your branch, and then I could do the renamings... would it be fine like that? Or is it another issue? I am not sure about how git does merging.

Reply:
It's not necessarily about the original file. Even without that, you might run into issues because git's merge heuristics are just not very good across different file names. I'd just not take that risk just to remove `py` from a filename...

Reply:
Fine, I will create another branch, and try to merge `simplify` to that, what happens

Reply:
https://github.com/IBM/graph2tac/pull/101